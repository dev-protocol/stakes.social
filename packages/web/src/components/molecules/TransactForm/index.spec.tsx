import React from 'react'
import Web3 from 'web3'
import { render } from '@testing-library/react'
import { TransactForm } from '.'
import 'src/__mocks__/window/matchMedia.mock'
import WalletContext from 'src/context/walletContext'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('src/fixtures/uniswap/hooks')
jest.mock('src/fixtures/wallet/hooks')
jest.mock('web3')
jest.mock('@dev/graphql')

describe(`${TransactForm.name}`, () => {
  test('Snapshot', () => {
    const component = render(
      <WalletContext.Provider value={{ web3: new Web3(), setWeb3: () => {} }}>
        <TransactForm onClickMax={() => {}} propertyAddress="" />
      </WalletContext.Provider>
    )
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
