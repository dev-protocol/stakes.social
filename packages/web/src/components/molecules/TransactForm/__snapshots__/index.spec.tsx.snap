// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactForm Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-1d8aayd-1 bcBbeW"
      style="opacity: 1;"
    >
      <button
        class="ant-btn ant-btn-lg sc-1ho13tb-0 sc-1d8aayd-2 cpPLtK gRTWBx"
        type="button"
      />
    </div>
  </div>
</body>
`;
